pkg/maps/lbmap/lbmap.go

```diff
@@ -216,7 +223,9 @@ func UpdateRevNat(key RevNatKey, value RevNatValue) error {
 }
 
 func DeleteRevNat(key RevNatKey) error {
-	log.Debugf("deleting RevNatKey: %s", key.String())
+	log.WithFields(log.Fields{
+		"revNATK": key,
```

Comment:
`UpdateRevNat` seems to use `logfields.BPFMapKey`, any reason not to use it here as well?

Reply:
Nope! I missed it when I decided have a global BPFMapKey. Fixed